prow/cmd/tide/main.go

```diff
@@ -96,6 +103,8 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.StringVar(&o.historyURI, "history-uri", "", "The /local/path,gs://path/to/object or s3://path/to/object to store tide action history. GCS writes will use the default object ACL for the bucket")
 	fs.StringVar(&o.statusURI, "status-path", "", "The /local/path, gs://path/to/object or s3://path/to/object to store status controller state. GCS writes will use the default object ACL for the bucket.")
 	// Gerrit-related flags
+	fs.BoolVar(&o.githubOnly, "github-only", false, "Use only when tide configuration contains both GitHub and Gerrit repos set")
```

Comment:
It would help to explain what this flag actually does, and not just explain under what conditions this flag is valid.

Reply:
done